tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -148,6 +148,7 @@ class CloudGov(Common):
             'connection_class': RequestsHttpConnection,
         },
     }
+    ELASTIC_INDEX_PREFIX = f'{APP_NAME}_'
```

Comment:
Just for my clarity, this will resolve to: `raft_` when the `raft` app is deployed right?

Reply:
correct

Reply:
note: our app names are being deployed with a prefix, so the APP_NAME will be "tdp-backend-raft"